cmd/root.go

```diff
@@ -74,6 +75,21 @@ func initConfig() {
 		// Search config in home directory with name ".config/rvault/config.yaml".
 		viper.AddConfigPath(home + "/.config/rvault")
 		viper.SetConfigName("config")
+
+		// Read ~/.vault-token file
+		tokenFile := home + "/.vault-token"
+		if _, err = os.Stat(tokenFile); !os.IsNotExist(err) {
+			fh, err := os.Open(tokenFile)
+			if err != nil {
+				klog.Exitf("Can't read token from file '%s': %v", tokenFile, err)
+			}
+			defer fh.Close()
+
+			s := bufio.NewScanner(fh)
+			if s.Scan() {
+				viper.SetDefault("global.token", s.Text())
+			}
+		}
```

Comment:
Yes, you're right! I like the third option, but who's gonna get the highest priority: `config.yml` or `.vault-token` file?

Reply:
> Yes, you're right! I like the third option

By these steps I was actually referring to the new block order (it was not a choice I mean) where we first find the home directory, we have an if..else to use a configuration if provided or the default location otherwise and finally a third block where we read vault-token.


> but who's gonna get the highest priority: `config.yml` or `.vault-token` file?

From what I read, `vault-token` is the result of a vault login right through the CLI right? I'd say config takes preference because it's specific to `rvault`. If you wish to use CLI tokens leave it blank and `vault-token` should be the one we use as "default" if the file exists.

Anyway, maybe this clarification is worth in the `README.md` (precedence) since it's not obvious at all.

Reply:
Oh, my bad! Got it now, please have a look at my latest commit.
Thank you!